MacOSSwiftUI/MacOSSwiftUI/ContentView.swift

```diff
@@ -21,7 +21,7 @@ struct ContentView: View {
             List {
                 ForEach(items) { item in
                     NavigationLink {
-                        Text("Item located at \(item.timestamp!, formatter: itemFormatter)").accessibilityHint("This is some extra Info")
+                        Text("Item located at \(item.timestamp!, formatter: itemFormatter)").accessibilityHint("This is some ")
```

Comment:
Please add something meaningful info for the hint

Reply:
I have added an meaningful hint